src/module/system/statistic/index.ts

```diff
@@ -503,8 +504,10 @@ class StatisticDifficultyClass {
             data.dc?.domains ? extractModifiers(parent.actor.synthetics, data.dc.domains) : [],
         ].flat();
         this.modifiers = allDCModifiers.map((modifier) => modifier.clone({ test: this.options }));
+    }
 
-        this.value = (data.dc?.base ?? 10) + new StatisticModifier("", this.modifiers, this.options).totalModifier;
+    get value(): number {
+        return 10 + new StatisticModifier("", this.modifiers, this.options).totalModifier;
```

Comment:
I fully agree with killing data.dc?.base (If we keep base, it should be to add a base modifier, not to simply add 10) but if I recall you prefer to keep these as values computed during initialization, not as getters.

Reply:
Ah wait no, we can't remove dc.base yet. That's used by spellcasting to set the base to 0 because we don't yet have a solution for a hardcoded base dcs.